docs/howto-guides/release-process.md

```diff
@@ -15,16 +15,14 @@ released? Here's how.
      [our python-app.yml GitHub
      action](https://github.com/microbiomedata/nmdc-runtime/blob/main/.github/workflows/python-app.yml),
      which is triggered automatically when a change to any Python file in the repository is pushed to the
-     `main` branch, or to a Pull Request. You can monitor the status of Github Actions
+     `main` branch, or to a Pull Request. You can monitor the status of GitHub Actions
      [here](https://github.com/microbiomedata/nmdc-runtime/actions).
 
-2. Add a summary for the release to `RELEASES.md`. You can make an edit and push to the `main`
-   branch [via GitHub](https://github.com/microbiomedata/nmdc-runtime/blob/main/RELEASES.md). This will
-   trigger two GitHub actions in sequence to
+2. Create a new [GitHub Release](https://github.com/microbiomedata/nmdc-runtime/releases). This will trigger GitHub Actions which:
```

Comment:
add some qualification like "tag it by typing 'v' and then using the autocomplete to fill in a release tag of the form 'vX.Y.Z', where if the release is a patch (fixing/refactoring something without changing behavior), increment Z by 1 relative to the last-release tag, if the release is minor (adds some new behavior without changing existing behavior), increment Y by 1 and set Z to 0, or if the release is major (old behavior has changed, so may break existing consumer workflows), increment X by 1 and set both Y and Z to 0."?

Reply:
Done.